org.eclipse.buildship.ui/src/main/java/org/eclipse/buildship/ui/wizard/project/ProjectPreviewWizardPage.java

```diff
@@ -376,14 +391,21 @@ private void populateTree(final OmniGradleBuildStructure buildStructure) {
 
                 @Override
                 public void run() {
```

Comment:
if (!ProjectPreviewWizardPage.this.projectPreviewTree.isDisposed()) {
  return;
}

--> less lines changed